src/Moq/Properties/Resources.resx

```diff
@@ -1,348 +1,351 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <root>
-  <!-- 
-    Microsoft ResX Schema 
```

Comment:
I am sorry to have to bother you again about whitespace... but I'm afraid I need to ask you again to revert the whitespace from tabs back to spaces so we see the actual changes to this file. 🙈

(This is my bad&mdash;after the first review of this PR, I adjusted this project's `.editorconfig` file such that it matches what VS autogenerates, so that this annoying problem would go away in the future. I should have waited with that until after this PR gets merged, then fixed up the whitespace myself. I'd like to apologise for that.)

Reply:
My git diff was ignoring whitespace/tab changes. Didn't notice there was such a problem... sorry too!